_posts/2016-06-15-beyond-progressive-web-apps-part-2.md

```diff
@@ -0,0 +1,171 @@
+---
+
+title: 'Beyond Progressive Web Apps Part 2: We Need to Know What’s New'
+author: Jan
+layout: blog-post
+comments: true
+
+---
+
+This is part two of a three part series about data sync beyond Progressive Web Apps. [Please read part one, before continuing](http://hood.ie/blog/beyond-progressive-web-apps-part-1.html "Beyond Progressive Web Apps Part 1").
+
+Say we have a news app that runs on mobile devices and a server that publishes new stories. Blogs and [RSS](https://en.wikipedia.org/wiki/RSS) might be a good real-world example.
+
+The scenario is this: our app starts for the first time and there are no stories available for the user to read on the device. So, the app asks the server to send the latest set of stories to the device. End result:  our users get to read some stories.
+
+Later, when our apps starts for the second time, your app asks for the latest set of stories. 
+
+And here is the first interesting bit: *we don’t want to send the app the stories it already has.*
+
+Why?
+
+Four reasons:
+
+1. They already are on the device, so it's redundant
```

Comment:
"They are already on the device" reads better than "They already are on the device"